### PR TITLE
editied footer - removed node branding (powered by)

### PR DESCRIPTION
**Pull request by abdullaalsulaiti**
removed the footer and nodebb powered by

### CODE REVIEW

File: install/data/footer.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
 	{
 		"widget": "html",
 		"data" : {
-			"html": "<footer id=\"footer\" class=\"container footer\">\r\n\t<div>\r\n\t\tPowered by <a target=\"_blank\" href=\"https://nodebb.org\">NodeBB</a> | <a target=\"_blank\" href=\"//github.com/NodeBB/NodeBB/graphs/contributors\">Contributors</a>\r\n\t</div>\r\n</footer>",
+			"html": "<footer id='footer' class='container footer'></footer>",
```
Comment (Inverseit) [inline install/data/footer.json:5]: Can you write some text here? Something like "NodeBB extended by Nirvana"? It is up to you, tho

**Review by Inverseit — CHANGES_REQUESTED** [review]

**Comment by dananaimi** [comment]
sc

**Comment by abdullaalsulaiti** [comment]
sc

**Review by Inverseit — APPROVED** [review]
Works

**Pull request merged**